Automattic/wp-calypso
# Extensions: WPJM - Detect whether or not to show setup wizard

This PR detects whether or not the WPJM setup wizard should be shown when a user navigates to the settings. It's modelled after the [`ExtensionRedirect`](https://github.com/Automattic/wp-calypso/tree/master/client/blocks/extension-redirect) block.

## Testing

1. If you haven't already, follow p6r3EZ-ra-p2 to install the WPJM plugin.
2. Log into WP Admin for your sandbox and activate the plugin.
3. `ssh` into your sandbox and `cd public_html`.
4. Enter `wp transient set _job_manager_activation_redirect 1 3600`.
5. In Calypso, go to the "Manage Plugins" page and select WP Job Manager.
6. Click on the "Edit plugin settings" link.
7. You should now be redirected to the setup wizard page.
